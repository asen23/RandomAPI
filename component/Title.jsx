export default function Title(props) {
    return <h1 className="text-4xl font-bold">{props.children}</h1>
}