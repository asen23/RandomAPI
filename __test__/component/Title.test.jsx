import { render, screen } from "@testing-library/react"
import userEvent from "@testing-library/user-event"
import Title from "component/Title"

describe("Title", () => {
    it("renders unchanged", async () => {
        const { asFragment } = render(<Title>A Title</Title>)
        expect(asFragment()).toMatchSnapshot()
    })

    it("renders a heading", () => {
        render(<Title>A Title</Title>)

        const heading = screen.getByRole("heading", {
            name: "A Title",
        })
        expect(heading).toBeInTheDocument()
    })
})
