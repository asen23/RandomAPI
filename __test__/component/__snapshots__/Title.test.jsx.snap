// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders unchanged 1`] = `
<DocumentFragment>
  <h1
    class="text-4xl font-bold"
  >
    A Title
  </h1>
</DocumentFragment>
`;
